spec/lib/declare_schema/model/column_spec.rb

```diff
@@ -63,7 +60,6 @@
     end
 
     describe '.deserialize_default_value' do
```

Comment:
It's hard to tell for sure, but I think the tests inside this block need to un-tabbed since the `if` was deleted